src/inet/InetInterface.h

```diff
@@ -390,6 +390,13 @@ class InterfaceIterator
 class DLL_EXPORT InterfaceAddressIterator
 {
 public:
+    enum class Flags : uint8_t
+    {
+        kNotFinal   = (1 << 0), // Not yet valid: Optimistic/DAD failed/tentative
```

Comment:
This seems like it's exposing Linux flags in an API that is mean to work across a range of platforms that may not have these granular flags.

Perhaps start with just one (since all of the logic below seems to be treating all of these the same), and expand in the future as needed?

Reply:
The flags here correspond to RFC 8981 concepts, so I think they're universally meaningful (even if not exposed on all platforms).